### ISSUE TITLE
X is already defined as X

### ISSUE THREAD

**Issue by francisdb** [swoval/MacOSXWatchService#36]
With `1.2.0-RC1` and sbt `1.1.0` I'm seeing a lot of these:

```
app/util/shared/PlayJsonExt.scala:6:8: PlayJsonExt is already defined as object PlayJsonExt
[error] object PlayJsonExt {
```
(introduced somewhere after `1.1.4`, see also discussion at #19)

Makes this plugin unusable

**Comment by eatkins** [comment]
I was able to reproduce the issue in a play project. My project now works with 1.2.0-RC2. Hopefully that will fix your issue as well.

**Comment by francisdb** [comment]
I can confirm this fixed in `1.2.0-RC3`
